### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by UniquelyAbbi** [cooperrc/computational-mechanics#17] [General]
Hello everyone, my name is Abbigale Whitmore. An interesting fact about me is that I enjoy learning new hobbies even if I am not good at them.

I am here because I am a Mechanical Engineering major and this is course is a requirement. I am sure this course will introduce me to skills I will need in the future which makes it helpful in my future career.

Jokingly I would say remove coding because I personally find it difficult, however I think that there is nothing I would change since a challenge is good. If anything I'd change the time constraint of having it in the summer.

**Comment by tempcooper** [comment]
> Jokingly I would say remove coding because I personally find it difficult

:rofl:  Let's change that! There are almost no engineering solutions that don't have digital controls/sensors/communication. We don't have to be database experts, but if we can write a `for`-loop and build some basic functions we can help make smart technology actually smart
